docs/tutorial.md

```diff
@@ -1,28 +1,31 @@
-# Hello World Tutorial
+# Tekton Pipelines Tutorial
 
-Welcome to the Tekton Pipeline tutorial!
+This tutorial uses a simple `Hello World` example to show you how to:
+- Create a `Task`,
+- Create a `Pipeline` containing your `Tasks`,
+- Use a `TaskRun` to instantiate and execute a `Task` outside of a `Pipeline`,
+- Use a `PipelineRun` to instantiate and run a `Pipeline` containing your `Tasks`.
 
-This tutorial will walk you through creating and running some simple
-[`Tasks`](tasks.md) & [`Pipelines`](pipelines.md) and running them by creating
-[`TaskRuns`](taskruns.md) and [`PipelineRuns`](pipelineruns.md).
+This tutorial consists of the following sections:
 
-- [Creating a hello world `Task`](#task)
-- [Creating a hello world `Pipeline`](#pipeline)
+- [Creating and running a `Task`](#creating-and-running-a-task)
```

Comment:
I actually prefer the original phrasing of `a hello world`, but perhaps what could be done here is to actually describe in greater detail what the task/pipeline do? 

I think the first Task executed independent of the pipeline is literally a hello world task and the next one is a Task that builds an image and is also used for the example Pipeline.

Reply:
I don't think we need to get this specific in the section titles. The code is self-explanatory.

Reply:
On second time through, I think this is actually fine. Sorry.